pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -573,6 +573,13 @@ func (s *Service) GetSnapshot(ctx context.Context, query cloudmigration.GetSnaps
 			s.log.Error("error applying plugin warnings, please open a bug report: %w", err)
 		}
 
+		// Log the errors for resources with errors at migration
+		for _, resource := range resources {
+			if resource.Status == cloudmigration.ItemStatusError && (resource.ErrorCode != "" || resource.Error != "") {
```

Comment:
We are checking if the `resource.ErrorCode != ""` but we are not using it. Is it intentional?